### PR TITLE
Implement ignoring a strange TRAVIS fail

### PR DESCRIPTION
**Pull request by ehsteve**
An issue is still open so we will not forget about this later. This is enable us to move forward on 0.6.

### CODE REVIEW

**Comment by Cadair** [comment]
@ehsteve can you point this at the 0.6 branch?

**Comment by Cadair** [comment]
:+1:

**Comment by dpshelio** [comment]
Am I missing something? How a bunch of blank lines fixes a problem?

File: sunpy/spectra/tests/test_callisto.py [hunk 1]
```diff
@@ -94,6 +96,7 @@ def test_query_number():
 
 
 @pytest.mark.online
+@pytest.mark.xfail
```
Comment (dpshelio) [inline sunpy/spectra/tests/test_callisto.py]: Oh! I see it now!! forget my previous comment.

**Comment by dpshelio** [comment]
:+1:

**Comment by ayshih** [comment]
#1383 is the clean version of this PR

**Pull request closed without merging**